qiskit_bot.yaml

```
@@ -0,0 +1,35 @@
+---
+notifications:
+    ".*":
+        - "@Qiskit/terra-core"
+    "visualization/pulse_v2":
+        - "@nkanazawa1989"
+    "qpy":
+        - "@mtreinish"
+        - "@nkanazawa1989"
+    "two_qubit_decompose":
+        - "@levbishop"
+    "quantum_info":
+        - "@ikkoham"
+    "utils/run_circuits":
+        - "@manoelmarques"
+        - "@woodsp-ibm"
+    "utils/quantum_instance":
+        - "@manoelmarques"
+        - "@woodsp-ibm"
+    "opflow":
+        - "@manoelmarques"
+        - "@woodsp-ibm"
+        - "@Cryosis"
```

here, and elsewhere - Julien's username is Cryoris (-ris not -sis)
```suggestion
        - "@Cryoris"
```